boards/dwm1001/Kconfig

```diff
@@ -17,4 +17,7 @@ config BOARD_DWM1001
     select HAS_PERIPH_UART
     select HAS_VDD_LC_FILTER_REG1
 
+config MODULE_SAUL_DEFAULT
+    select MODULE_LIS2DH12
```

Comment:
I think this should be done via selecting features.

Reply:
Seems like you are proposing to take in Kconfig the path I wanted to take in https://github.com/RIOT-OS/RIOT/pull/12952

Reply:
Something like done here https://github.com/RIOT-OS/RIOT/pull/16719/commits/537c4c7096c11c78e6f54cf4de0d0c9cbc990e5c#diff-4c34f84db50e5629afa528edd6075841aaaa672bf04c520c2b432d77d77757b9R8-R30